src/index.ts

```diff
@@ -1,92 +1,81 @@
 import cluster from "node:cluster";
-import http from "node:http";
+import { Bot } from "grammy";
+import dotenv from "dotenv";
 
-const PORT = 3000;
+dotenv.config({ path: ".env.local" });
+if (!process.env.BOT_TOKEN_ONE) {
+  throw new Error("BOT_TOKEN_ONE is not defined");
+}
+if (!process.env.BOT_TOKEN_TWO) {
+  throw new Error("BOT_TOKEN_TWO is not defined");
+}
 
-if (cluster.isPrimary) {
-  console.log(`Primary ${process.pid} is running`);
+interface WorkerMessage {
```

Comment:
We can try to make invalid states unrepresentable here. Something like:
```typescript
type WorkerMessage = InitializeBotMessage | ShutdownMessage;
type InitializeBotMessage = { type: 'INITIALIZE_BOT', token: string };
type ShutdownMessage = { type: 'SHUTDOWN' };
```
This way, we won't have a message with the type SHUTDOWN and a non-empty token.